conformance/README.md

```diff
@@ -561,12 +601,14 @@ type.
 These rules describe the overall shape of test cases:
 
 ```ebnf
-test ::=  "("  "ion_1_0"  fragment*  continuation  ")"
+test ::=  "("  "document"            continuation  ")"
+       |  "("  "ion_1_0"  fragment*  continuation  ")" 
        |  "("  "ion_1_1"  fragment*  continuation  ")"
        |  "("  "ion_1_x"  fragment*  continuation  ")"
```

Comment:
How important is it to retain all these? I'm mostly indifferent right now, but if there were any more, I think it would be too many ways to express the same thing.

Reply:
Since the expansions here are non-trivial (especially `ion_1_x`), I think these are valuable.

I don't see this as many ways to express the same thing, I see this as abstraction over the most common patterns in the test cases, to reduce bulk and potential for error.